website/docs/docs/cloud/manage-access/about-access.md

```diff
@@ -8,142 +8,203 @@ pagination_prev: null
+You can edit an existing user's license by navigating to the **Users** section of the **Account settings**, clicking on a user, and clicking **Edit** on the user pane.
 
-For more information on these license types, see [Seats & Users](/docs/cloud/manage-access/seats-and-users).
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/edit-user.png" width="60%" title="Example of the user information window in the user directory" />
 
-## Role-based access control
+Delete users from this same window to free up licenses for new users.
 
-:::info dbt Cloud Enterprise
+### Groups
+
+Groups in dbt Cloud serve much of the same purpose as they do in traditional directory tools &mdash; to gather individual users together to make bulk assignment of permissions easier. Admins use groups in dbt Cloud to assign [licenses](#licenses) and [permissions](#permissions). The permissions are more granular than licenses, and you only assign them at the group level; _ you can’t assign permissions at the user level._ Every user in dbt Cloud must be assigned to at least one group.
+
+There are three default groups available as soon as you create your dbt Cloud account (the person who created the account is added to all three automatically):
+
+- **Owner:** This group is for individuals responsible for the entire account and will give them elevated account admin privileges. You cannot change the permissions. 
+- **Member:** This group is for the general members of your organization, who will also have full access to the account. You cannot change the permissions. By default, dbt Cloud adds new users to this group.
+- **Everyone:** A general group for all members of your organization. Customize the permissions to fit your organizational needs. By default, dbt Cloud adds new users to this group.
+
+We recommend deleting the default `Owner`, `Member`, and `Everyone` groups before deploying and replacing them with your organizational groups. This prevents users from receiving more elevated privileges than they should and helps admins ensure they are properly placed.
+
+Create new groups from the **Groups & Licenses** section of the **Account settings**. If you use an external IdP for SSO, you can sync those SSO groups to dbt Cloud from the **Group details** pane when creating or editing existing groups.
+
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/new-group.png" width="60%" title="Example the new group pane in the account settings." />
 
-Role-based access control is a feature of the dbt Cloud Enterprise plan
+If a user is assigned licenses and permissions from multiple groups, the group that grants the most access will take precedence. You must assign a permission set to any groups created beyond the three defaults, or users assigned will not have access to features beyond their user profile.
```

Comment:
I think we should keep this specific because groups can be assigned with no permissions. It's about license and permission assignments. One piece of feedback was that customers were getting confused, so I think leaving in the specific language will help. 

Maybe making it a callout would work better? This is an important part to stress

Reply:
> Maybe making it a callout would work better? This is an important part to stress

That might work! See how it looks on the page. Your call either way!